### ISSUE TITLE
Audio zones: Allow editing

### ISSUE THREAD

**Issue by JulianGro** [vircadia/vircadia#949]
Currently the UI doesn't allow editing, only adding and removing audio zones.
An edit button would save a lot of hassle and make the software not look completely unrefined.
![1Auswahl_810](https://user-images.githubusercontent.com/11144627/103395209-68469600-4b2d-11eb-9675-be03d2326fd1.png)

**Comment by Aitolda** [comment]
I agree this would be nice.  It's a quality of life improvement. Currently i have to recreate before i delete the old one.

**Comment by JulianGro** [comment]
yes :angry:
